### PR TITLE
chore: add pinned items service

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #

This adds the commands for yarn generate to also pull in the pinned items routes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,11 +52,11 @@
...
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
-    "generate": "export SHA=a29e24cb19c002132e978d78e6f39bee74ebec56 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
+    "generate": "export SHA=8e8ba92e4764cd8914284315fb87c1945c757497 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
     "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
     "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations-oss",
-    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations && yarn cloudPriv",
+    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations && yarn cloudPriv && yarn pinned",
```
Comment (Asalem1) [inline package.json:59]: godspeed. Just checking this won't be in OSS right?

File: package.json [hunk 2]
```diff
@@ -68,7 +68,8 @@
     "annotations": "oats ${REMOTE}contracts/priv/annotationd.yml > ./src/client/annotationdRoutes.ts",
     "annotations-oss": "oats ${REMOTE}contracts/priv/annotationd-oss.yml > ./src/client/annotationdRoutes.ts",
     "mapsd": "oats ${REMOTE}contracts/mapsd.yml > ./src/client/mapsdRoutes.ts",
-    "managedFunctions": "oats ${REMOTE}contracts/managed-functions.yml > ./src/client/managedFunctionsRoutes.ts"
+    "managedFunctions": "oats ${REMOTE}contracts/managed-functions.yml > ./src/client/managedFunctionsRoutes.ts",
+    "pinned": "oats ${REMOTE}contracts/priv/pinneditemsd.yml > ./src/client/pinnedItemRoutes.ts"
```
Comment (Asalem1) [inline package.json:72]: I trust that this exists on the SHA version of openapi

**Review by Asalem1 — APPROVED** [review]

**Review by genehynson — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ShmuelLotman) [reply package.json:59]: nope

File: package.json (hunk 2 above)
Comment (ShmuelLotman) [reply package.json:72]: you better believe it sweetie

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2164)

**Pull request merged**